code/api/src/modules/deliveryProduct/resolvers.js

```diff
@@ -0,0 +1,22 @@
+// App Imports
+import models from '../../setup/models'
+
+// Create DeliveryProduct
+export async function create(parentValue, { deliveryId, productId }) {
+  console.log('hello world')
+  return await models.DeliveryProduct.create({
```

Comment:
Lets check out what deliveryId and productId are. Are they coming in as an integer or an object?